### PR TITLE
Fix turbo build on Windows

### PR DESCRIPTION
**Pull request by CTNicholas**
`cp` doesn't work on Windows, but [shx](https://www.npmjs.com/package/shx) makes it cross-platform.

### CODE REVIEW

File: packages/liveblocks-yjs/package.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
   ],
   "scripts": {
     "dev": "tsup --watch",
-    "build": "tsup && cp dist/index.d.ts dist/index.d.mts",
+    "build": "tsup && shx cp dist/index.d.ts dist/index.d.mts",
```
Comment (nvie) [inline packages/liveblocks-yjs/package.json:27]: Hmm. This `cp` should no longer be necessary with the latest version of `tsup`. I'll have a look at the configuration and upgrade this.

Comment (nvie) [reply packages/liveblocks-yjs/package.json]: Fixed!

**Review by nvie — APPROVED** [review]

File: packages/liveblocks-yjs/package.json [hunk 2]
```diff
@@ -24,7 +24,7 @@
   ],
   "scripts": {
     "dev": "tsup --watch",
-    "build": "tsup && cp dist/index.d.ts dist/index.d.mts",
+    "build": "tsup",
```
Comment (nvie) [inline packages/liveblocks-yjs/package.json:27]: This was necessary in an older version of tsup, so we could simply remove this — sorry about the hassle, and thanks for catching! 👍

Comment (CTNicholas) [reply packages/liveblocks-yjs/package.json:27]: Thanks for making it simpler!

**Pull request merged**